packages/components/psammead-brand/src/SkipLink/index.jsx

```diff
@@ -0,0 +1,49 @@
+import styled, { css } from 'styled-components';
+import { oneOf } from 'prop-types';
+import { GEL_GROUP_2_SCREEN_WIDTH_MAX } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const SKIP_LINK_COLOR = '#333';
+const SKIP_LINK_BORDER = '0.1875rem'; // 3px
+const TOP_BOTTOM_SPACING = '0.75rem'; // 12px
+
+const SkipLink = styled.a`
+  position: absolute;
+  clip-path: inset(100%);
+  clip: rect(1px, 1px, 1px, 1px);
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  padding: ${TOP_BOTTOM_SPACING} ${GEL_SPACING};
+  background-color: #ffffff;
+  border: ${SKIP_LINK_BORDER} solid #000;
+  color: ${SKIP_LINK_COLOR};
+  text-decoration: none;
+  ${({ script }) => script && getPica(script)};
+  ${({ service }) => service && getSansRegular(service)}
```

Comment:
Missing a ; on the end? `${({ service }) => service && getSansRegular(service)};`